core/src/commonMain/kotlin/com/ustadmobile/core/controller/SessionsListPresenter.kt

```diff
@@ -0,0 +1,60 @@
+package com.ustadmobile.core.controller
```

Comment:
Should be SessionListPresenter (we don't use plurals e.g. ClazzListPresenter, PersonListPresenter, etc)